src/index.js

```diff
@@ -108,6 +108,7 @@ function ParseServer({
     choosePassword: undefined,
     passwordResetSuccess: undefined
   },
+  sessionLength: sessionLength,
```

Comment:
you should not add sessionLengh as a second argument, but inside the spreaded option in this format:
sessionLength = 31536000000, // 1 year in ms

I don't be believe we need to be in ms, seconds would be enough